test/results/reconstruction-test.py

```diff
@@ -0,0 +1,124 @@
+AudioFormats = ["lilcom", "mp3_320", "mp3_256", "mp3_224", "mp3_192", "mp3_160",\
+     ]
+
```

Comment:
Thanks.  As a matter of good practice, please put this file through the pep8 style checker.